README.md

```diff
@@ -18,6 +18,10 @@
         <img alt="YouTube Channel Subscribers" src="https://img.shields.io/youtube/channel/subscribers/UCasf2Q7X8nF7S4VEmcTHJ0Q?style=social"></a>
 
 <!-- TODO: code coverage -->
+### Versatile Data Kit consists of two main components
+
+![versatile-data-kit-componenets](https://user-images.githubusercontent.com/11227374/229588579-7c8d1d51-206e-48d4-8957-a6c9d2190c94.svg)
```

Comment:
This might not be the best image to start the README. It conveys important information, but I don't think it's the absolute first thing users should see. Maybe it can have its own section below.

Reply:
Also, maybe you can add it in the repo and not use the github CDN. It's much easier to see the diff between images if you change it in the future.